h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -661,29 +661,39 @@ public static void main( String[] args ) {
         Chunk c0 = chks[cols                 ];
         Chunk c1 = chks[cols+(chks.length>>1)];
         for( int rows = 0; rows < chks[0]._len; rows++ ) {
-          if (c0 instanceof C16Chunk && c1 instanceof C16Chunk) {
-            if (! (c0.isNA(rows) && c1.isNA(rows))) {
+          if ((!(c0.isNA(rows)) && c1.isNA(rows)) || (c0.isNA(rows) && !(c1.isNA(rows)))) {
```

Comment:
I think it's more clear and easy to read  if you just say c0.isNA(i) != c1.isNA(i